command/genesis/params.go

```diff
@@ -142,12 +144,12 @@ func (p *genesisParams) validateFlags() error {
 		return errInvalidEpochSize
 	}
 
-	// Validate min and max validators number
-	if err := command.ValidateMinMaxValidatorsNumber(p.minNumValidators, p.maxNumValidators); err != nil {
-		return err
+	if _, err := os.Stat(p.validatorsPath); errors.Is(err, os.ErrNotExist) {
```

Comment:
Will this work with passing validators as param?

Reply:
Yes, it would work, due to the fact that the default value for `validatorsPath` is set to the current directory ("./").
The exact same code was present in the manifest command: https://github.com/0xPolygon/polygon-edge/pull/1389/files#diff-81dfae315a2b068d32b49fb15f77a51a35a3825447063099b021f800156b45f7L150.

However, I'm going to add an explicit check and validate this value only if `validators` are not provided via flag.

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1389/commits/63f163fbc16b21636aa10541ccb7ab99e8131fca